src/MatrixClientPeg.js

```diff
@@ -40,6 +40,14 @@ function deviceId() {
 class MatrixClientPeg {
     constructor() {
         this.matrixClient = null;
+
+        // These are the default options used when Lifecycle.js
+        // starts the client. These can be altered when the
+        // 'will_start_client' event is dispatched.
+        this.opts = {
```

Comment:
The reason it's here is because Lifecycle is pure functions, so I wanted to keep this here where we keep the other state.